gammapy/datasets/tests/test_simulate.py

```diff
@@ -551,14 +551,21 @@ def test_mde_run(dataset, models):
     assert meta["MMN00000"] == "test-bkg"
     assert meta["MID00001"] == 1
     assert meta["NMCIDS"] == 2
-    assert_allclose(float(meta["ALT_PNT"]), float("-13.5345076464"), rtol=1e-7)
-    assert_allclose(float(meta["AZ_PNT"]), float("228.82981620065763"), rtol=1e-7)
+    assert_allclose(meta["ALT_PNT"], -13.5345076464, rtol=1e-7)
+    assert_allclose(meta["AZ_PNT"], 228.82981620065763, rtol=1e-7)
     assert meta["ORIGIN"] == "Gammapy"
     assert meta["TELESCOP"] == "CTA"
     assert meta["INSTRUME"] == "1DC"
     assert meta["N_TELS"] == ""
     assert meta["TELLIST"] == ""
 
+    # test writing out and reading back in works
+    obs.events = events
+    path = tmp_path / "obs.fits.gz"
+    obs.write(path)
+    obs_back = Observation.read(path)
+    assert obs_back.observatory_earth_location == LOCATION
```

Comment:
This test is failing in the CI. It seems that the __eq__ is not implemented for the class `ObservationLocation`.